src/js/rx/components/SortMenu.jsx

```diff
@@ -21,35 +21,45 @@ class SortMenu extends React.Component {
     this.props.onChange(event.target.value);
   }
 
-  handleClick(sortValue) {
+  handleClick(sortValue, sortOrder) {
     return (event) => {
       event.preventDefault();
-      this.props.onClick(sortValue);
+      this.props.onClick(sortValue, sortOrder);
     };
   }
 
-  render() {
-    const sortBys = this.props.options;
-    const sortLinks = (options) => {
-      return options.map((o, ind) => {
-        // Sets an rx-sort-active class when the option value
-        // matches the selected / default value
-        const selectedClass = classNames({
-          'rx-sort-active': o.value === this.props.selected
-        });
+  renderSortLinks() {
+    const { selected: { value, order } } = this.props;
 
-        return (
-          <li key={ind}>
-            <a
-                className={selectedClass}
-                onClick={this.handleClick(o.value)}>
-              {o.label}
-            </a>
-          </li>
-        );
+    const options = ['refillSubmitDate',
+      'lastSubmitDate',
+      'refillDate',
+      'refillStatus'
```

Comment:
Should `refillStatus` be associated with 'Newest to Oldest'/'Oldest to Newest'? Seems 'A-Z'/'Z-A' would be more appropriate, although there is that tricky issue of the display names not falling into exact alphabetical order.